src/entity/CommentReport.ts

```diff
@@ -0,0 +1,28 @@
+import { Column, Entity, Index, ManyToOne, PrimaryColumn } from 'typeorm';
+import { User } from './User';
+
+@Entity()
+export class CommentReport {
+  @PrimaryColumn({ type: 'text' })
+  @Index('IDX_comment_report_comment_id')
+  commentId: string;
+
+  @PrimaryColumn({ length: 36 })
+  @Index('IDX_comment_report_user_id')
+  userId: string;
+
+  @Column({ default: () => 'now()' })
+  createdAt: Date;
+
+  @Column({ length: 12 })
```

Comment:
We have a report reason that pertains to misinformation and has thought of using `MISINFORMATION` as its key. The limit is 12, but the said key is 14 character. We can maybe extend the length to 16? WDYT?
```suggestion
  @Column({ length: 16 })
```

Reply:
Oh good catch, could even be safe and make it 36 doens't add much overhead I personally think.